### PR TITLE
tests/resource/workspaces_directory: Fix hardcoded region

### PR DESCRIPTION
**Pull request by YakDriver**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates #12995 [Phase 2](https://github.com/hashicorp/terraform-provider-aws/issues/12995#issuecomment-730680494)

Release note for [CHANGELOG](https://github.com/hashicorp/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note
NONE
```

### Acceptance Tests in GovCloud

```
--- SKIP: TestAccAwsWorkspacesDirectory_basic (2.36s)
```

### Acceptance Tests in `us-west-2`

```
--- PASS: TestAccAwsWorkspacesDirectory_basic (615.53s)
```

### Acceptance Tests in `us-east-1`

```
--- PASS: TestAccAwsWorkspacesDirectory_basic (911.57s)
```

### CODE REVIEW

File: aws/resource_aws_workspaces_directory_test.go [hunk 1]
```diff
@@ -515,7 +516,7 @@ data "aws_availability_zones" "available" {
 
 locals {
   region_workspaces_az_ids = {
```
Comment (Tensho) [inline aws/resource_aws_workspaces_directory_test.go:518]: @YakDriver The basic idea behind this map providing exact AZs (with ids), where the WorkSpaces service is available. If you need to test workspace directory resource in Gov region simply add extra mapping of region name and respective AZ ids. Then `lookup` function will pick up it according to the region, where the test is run.

**Review by bflad — DISMISSED** [review]
In this case, I think lint ignoring is fine and we should prefer to keep the region name in the configuration for clarity, since it looks like the `local` is written to be expanded in the future. 👍  (I have not done any research to see if there is a better solution to the "acceptable AZs for WorkSpaces" problem.)

**Comment by YakDriver** [comment]
@bflad I try to avoid lint ignoring big blocks of config since offending stuff can be inserted anywhere in the config. But, not sure what else to do here. It's "fixed."

**Review by bflad — APPROVED** [review]
Looks good to me, (although the test on this branch is failing for unrelated reasons fixed on the main branch).

Output from acceptance testing in AWS Commercial:

```
=== CONT  TestAccAwsWorkspacesDirectory_basic
    resource_aws_workspaces_directory_test.go:63: Step 1/2 error: Check failed: 1 error occurred:
        	* Check 16/24 error: aws_workspaces_directory.main: Attribute 'tags.%' expected "0", got "1"
```

Output from acceptance testing in AWS GovCloud (US):

```
--- SKIP: TestAccAwsWorkspacesDirectory_basic (2.26s)
```

**Pull request merged**